lua/luasnip/loaders/init.lua

```diff
@@ -37,6 +57,11 @@ function M.edit_snippet_files(opts)
 		return {}
 	end
 
+	if opts.target_snippet then
+		edit_specific_snippet(opts.target_snippet, edit)
+		return
+	end
+
```

Comment:
I think this edit specific snippet-funcitonality does not really fit in here: `edit_snippet` has this choice, which doesn't really make sense if all wants to do is edit a specific snippet. How about putting something like this in `extras`?

Reply:
Sure!